test/unit/default_responses.js

```diff
@@ -370,28 +370,3 @@ exports['add response when recipient is allowed'] = function (test) {
         test.done();
     });
 };
-
-exports['add response with denied state when recipient is denied'] = function (test) {
```

Comment:
because the code in `default_responses.js` actually doesn't do that behavior o_O. 
It's the `messages.js` that does the `denied` status, and it's not specific to default responses anyway. The test should have been broken, it was testing something impossible, but it was passing because it was using a spy instead of a mock.